### ISSUE TITLE
Update Social Embed related CSP directives

### ISSUE THREAD

**Issue by simonsinclair** [bbc/simorgh#6335]
### Is your feature request related to a problem? Please describe.
We recently came across a Tweet in Kyrgyz that triggered new CSP errors, which need to be resolved.

- Run: `npm run build && npm start`
- Visit: http://localhost:7080/kyrgyz/23292889

#### Social Embed related errors
```
Refused to load the stylesheet 'https://ton.twimg.com/tfw/css/syndication_bundle_v1_73385286cca9d2256f6bf3993470820d4827b058.css' because it violates the following Content Security Policy directive: "style-src https://news.files.bbci.co.uk https://platform.twitter.com 'unsafe-inline'". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.

Refused to load the image 'https://ton.twimg.com/tfw/css/syndication_bundle_v1_73385286cca9d2256f6bf3993470820d4827b058.css' because it violates the following Content Security Policy directive: "img-src https://ichef.bbci.co.uk https://ichef.test.bbci.co.uk https://ping.chartbeat.net https://a1.api.bbc.co.uk/hit.xiti https://logws1363.ati-host.net https://news.files.bbci.co.uk https://news.test.files.bbci.co.uk https://*.akstat.io https://r.bbci.co.uk https://pagead2.googlesyndication.com https://securepubads.g.doubleclick.net https://tpc.googlesyndication.com https://www.google.com https://via.placeholder.com http://b.files.bbci.co.uk http://ping.chartbeat.net https://syndication.twitter.com https://platform.twitter.com https://pbs.twimg.com https://i.ytimg.com data: 'self'".
```

### Describe the solution you'd like

Prerequisites: Mozart dev. environment.

- Update Simorgh CSP
- Update the CSP following: https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/updating-csp.md.
- Ensure that any stale origins are removed.
- Compile a list of assets types that might raise issues (e.g. video may cause an issue)

#### Useful PRs:
https://github.com/bbc/simorgh/pull/6209
https://github.com/bbc/mozart-routing/pull/1307

### Describe alternatives you've considered
n/a

### Testing notes

Test with many tweet types e.g. audio, video. 

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

**Comment by oluoluoxenfree** [comment]
Blocked pending social embed asset being merged into Simorgh. Ask @simonsinclair

**Comment by simonsinclair** [comment]
Unblocked following the merge of https://github.com/bbc/simorgh/pull/6345.

**Comment by FK78** [comment]
Blocked until https://github.com/bbc/simorgh-infrastructure/issues/987 is resolved.
